.eslintrc.js

```diff
@@ -142,6 +142,15 @@ module.exports = {
       'asc',
       { ignoreClassNames: false, ignoreStyleProperties: false },
     ],
+
+    // Sort props for JSX component
+    'react/jsx-sort-props': [
+      'warn',
```

Comment:
Why `warn` and not `error`? In my experience warnings are easy to ignore and they will add up to so many that you stop noticing them.

Reply:
agree, did not want to be too strict)

Reply:
Agreed!